app/jobs/audio_processing.rb

```diff
@@ -4,6 +4,7 @@ class AudioProcessing < ActiveJob::Base
   @queue = :medium
 
   def perform(opts)
+    Apartment::Tenant.switch!(opts[:tenant])
```

Comment:
Why use `switch!` instead of `switch` and wrap everything below in a block?

Reply:
No specific reason, we can change to that approach.